dotCMS/src/integration-test/java/com/dotmarketing/util/HashBuilderTest.java

```diff
@@ -0,0 +1,98 @@
+package com.dotmarketing.util;
+
+import com.google.common.hash.Hashing;
+import com.liferay.util.Encryptor;
+import com.liferay.util.HashBuilder;
+import org.junit.Test;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.InputStream;
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.security.MessageDigest;
+import java.util.Base64;
+
+public class HashBuilderTest {
+
+    @Test
+    public void test_buildBytes () throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5547873)